Sources/PaymentsCore/Networking/Enums/NetworkingError.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+// TODO: Refine how we want to do error handling
+/// For further reading, see https://developer.apple.com/documentation/swift/error
+/// For Discussion:
+/// Error conforming enum vs Error conforming Struct?
+public enum NetworkingError: Error {
+    case networkingError(Error)
```

Comment:
This name isn't clear, since it's the same as the class. What about `connectionIssue`